vision/api/Detect/Program.cs

```diff
@@ -251,6 +256,23 @@ private static object DetectText(Image image)
             return 0;
         }
 
+        private static object DetectTextWithLocation(Image image)
+        {
+            // [START vision_set_endpoint]
+            // Instantiate a client connected to the 'eu' location.
+            var client = new ImageAnnotatorClientBuilder()

```

Comment:
nit: Remove the (), they are not needed on object initializers.

Reply:
Did not know that! Thank you.